src/multiCursor.ts

```diff
@@ -0,0 +1,19 @@
+import { Command, EditorView } from '@codemirror/view';
+import { EditorSelection } from '@codemirror/state';
+
+const addCursorFactory =
```

Comment:
nooooo not a factory haha we're not writing Java here 😆 

maybe something like `createAddCursor()`

Reply:
:shrug: